Customizing/global/plugins/Services/Repository/RepositoryObject/TalentAssessment/classes/Settings/ilDB.php

```diff
@@ -318,8 +318,10 @@ public function getVenueOptions() {
 	public function getOrgUnitOptions() {
 		$evg_id = \gevOrgUnitUtils::getEVGOrgUnitRefId();
 		$org_unit_utils = \gevOrgUnitUtils::getAllChildren(array($evg_id));
+		$evg_obj_id = \ilObject::_lookupObjId($evg_id);
+		$evg_title = \ilObject::_lookupTitle($evg_obj_id);
 
-		$ret = array();
+		$ret = array($evg_obj_id => $evg_title);
```

Comment:
Wunsch war es _EVG_ mit in der Auswahlliste zu haben. Vorher wurden nur die Kinder von _EVG_ hinzugefügt. Deshalb wurde das array leer initiiert. Jetzt mit den Daten zur _EVG_Auswahl.